opentasks/src/main/res/layout/task_editor_activity.xml

```diff
@@ -6,7 +6,7 @@
     <LinearLayout
             android:id="@+id/content"
             android:layout_width="match_parent"
```

Comment:
There is another `match_parent` in this line.

Reply:
idk, I assume Android Studio knows what it is doing. Probably this change would break something while the made change didn't.

Reply:
oh, right, I mixed it up with the `fill_parent` vs. `match_parent` issue.